experiments/tf_trainer/common/text_preprocessor.py

```diff
@@ -30,7 +30,7 @@ class TextPreprocessor():
   may include fixing a max word length.
   """
 
-  def __init__(self, embeddings_path: types.Path) -> None:
+  def __init__(self, embeddings_path: str) -> None:
```

Comment:
Why did we decide to get rid of the Path type? (not that I mind)

Reply:
Although I think it's good I just thought the trade off with convenience given that other libraries expected strings for paths made it not worth it.